src/service-worker.js

```diff
@@ -0,0 +1,17 @@
+workbox.core.setCacheNameDetails({prefix: 'goggl'});
```

Comment:
```suggestion
workbox.core.setCacheNameDetails({ prefix: 'goggl' });
```

スタイルガイドはAirbnbのでいくから、中括弧の内側はスペースを入れよう
https://mitsuruog.github.io/javascript-style-guide/#whitespace--in-braces